### ISSUE TITLE
missing lincense

### ISSUE THREAD

**Issue by petrkr** [xaionaro-go/trezorCipherKeyValue#3]
Hello,
becase I want use this package as base for initrd boot in NixOS, they request license or they will use restrictive `unfree`. For debian based systems it's ok, because you not need community to build custom packages.

https://github.com/NixOS/nixpkgs/pull/56551#discussion_r266188490

**Comment by xaionaro** [comment]
@petrkr thank you. Fixed. GPLv3. If you need it under another license then please let me know.

Please close the ticket if everything is OK :)

**Comment by petrkr** [comment]
Thanks, also u can check mine fork, i added somehow T support, but i had to also fork protobuf repo, maybe better do own implementation?

**Comment by xaionaro** [comment]
@petrkr: actually it seems @karel-3d is refactoring official trezor's code so `tesoro` will not be needed anymore (if he will succeed). I'd recommend to wait until he will finish.

Also the Trezor T should've work without the fork. I didn't test it in this utility, however I've added support of Trezor T into the `cryptoWallet` library in the end of 2018.

**Comment by petrkr** [comment]
Actually it does not work if u use passphrase, as workflow of messages is different.

**Comment by xaionaro** [comment]
> Actually it does not work if u use passphrase, as workflow of messages is different.

Well, yes. And I remember that I've implemented it within `cryptoWallet`. I've tested that the implementation works and was going to add a support of Trezor T to gocryptfs (but meanwhile I've relocated to another country and there left no time on this).

Anyway I think I will continue to work on this (to close all known issues) when/if @karel-3d will finish his work.
